extensions/wikia/WikiaApi/WikiaApiQueryDomains.php

```diff
@@ -85,6 +102,7 @@ public function execute() {
 		} else {
 			$this->addFields(array('city_id', 'city_url', 'city_lang'));
 			$this->addOption( "ORDER BY ", "city_id" );
+			$this->addOption( 'LIMIT', $limit );
```

Comment:
If the variable is not provided by user in the URL, MW API will set it to the default value declared on https://github.com/Wikia/app/pull/12627/files#diff-7f7f6f25d0e2d3ff0f4770cea8e296dbR165 . So without a provided parameter, limit will be 10.